src/clustering/sharding-sqlserver/SqlSharding.Host/Actors/ProductIndexActor.cs

```diff
@@ -64,15 +64,15 @@ protected override void PreStart()
         /*
          * Kicks off an Akka.Persistence.Query instance that will continuously
          */
-        var query = Context.System.ReadJournalFor<SqlReadJournal>(SqlReadJournal.Identifier);
-        query.PersistenceIds()
-            .Where(c => c.StartsWith(ProductTotalsActor.TotalsEntityNameConstant))
-            .Select(c =>
-            {
-                var splitPivot = c.IndexOf("-", StringComparison.Ordinal);
-                return new ProductFound(c[(splitPivot + 1)..]);
-            })
-            .To(Sink.ActorRef<ProductFound>(Self, Done.Instance))
-            .Run(Context.Materializer());
+        // var query = Context.System.ReadJournalFor<SqlReadJournal>(SqlReadJournal.Identifier);
```

Comment:
Disabled this query because otherwise it will cause the `ProductIndexActor` to query all of the entities, which prevents us from seeing of the remember-entities migration worked or not.